### PR TITLE
HDDS-4735. Fix typo in hdds.proto

### PR DESCRIPTION
**Pull request by symious**
## What changes were proposed in this pull request?

- Fix misleading type in hdds.proto

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4735

## How was this patch tested?

Manually test.

### CODE REVIEW

**Review by ayushtkn — APPROVED** [review]
+1

**Comment by symious** [comment]
@ayushtkn Thanks for the review.

File: hadoop-hdds/interface-client/src/main/proto/hdds.proto [hunk 1]
```diff
@@ -158,7 +158,7 @@ message NodePool {
 
 /**
  * LifeCycleState for SCM object creation state machine:
- *    ->Allocated: allocated on SCM but clean has not started creating it yet.
+ *    ->Allocated: allocated on SCM but client has not started creating it yet.
  *    ->Creating: allocated and assigned to client to create but not ack-ed yet.
  *    ->Open: allocated on SCM and created on datanodes and ack-ed by a client.
  *    ->Close: container closed due to space all used or error?
```
Comment (amaliujia) [inline hadoop-hdds/interface-client/src/main/proto/hdds.proto:164]: Nit: as this PR is fixing type, maybe remove the question mark of `Close: container closed due to space all used or error?`

Comment (symious) [reply hadoop-hdds/interface-client/src/main/proto/hdds.proto]: Thanks for the suggestion, have fixed the question mark.

**Comment by amaliujia** [comment]
+1 LGTM

**Review by cku328 — COMMENTED** [review]
Thanks @symious for the contribution.
Can you help correct the following two typos in the same file together?

https://github.com/apache/ozone/blob/9f344a9db692c1e7ebe52f4b182209cc20d24d64/hadoop-hdds/interface-client/src/main/proto/hdds.proto#L22
`a *unstable* .proto` -> `an *unstable* .proto`

https://github.com/apache/ozone/blob/9f344a9db692c1e7ebe52f4b182209cc20d24d64/hadoop-hdds/interface-client/src/main/proto/hdds.proto#L54
`transfered by ProtoBuf` -> `transferred by ProtoBuf`

**Comment by symious** [comment]
@cku328 Thanks for the comment. Have fixed the two typos.

**Review by cku328 — APPROVED** [review]

**Pull request merged**